package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby",
   "description": "React.js Static Site Generator",
-  "version": "0.6.2",
+  "version": "0.7.0",
```

Comment:
I would recommend against including version changes in PRs. If this gets superseded by another version update, you'll end up with a merge conflict and it will be harder for the maintainer to merge. Just a heads up! :smile: